app/src/component/AppNavigator.js

```diff
@@ -262,15 +265,37 @@ class AppWithNavigationState extends React.Component {
   }
 
   componentDidMount() {
+    this.emailVerifyCheckInterval = setInterval(() => this.checkEmailVerification(), 5000);
     Linking.addEventListener('url', this._handleUrl);
   }
 
+  checkEmailVerification = () => {
+    const { dispatch } = this.props;
+    AsyncStorage.getItem(Constants.KEY_EMAIL_VERIFY_PENDING).then(pending => {
```

Comment:
Can you use an `await` here?

Reply:
I suppose I could, but I would also need to add the `async` keyword to the function itself.